test/validation/api/scheduler/scheduler.c

```diff
@@ -988,6 +986,52 @@ static void scheduler_test_create_group(void)
 	CU_ASSERT_FATAL(odp_schedule(NULL, wait_time) == ODP_EVENT_INVALID);
 }
 
+static void scheduler_test_create_group(void)
+{
+	odp_thrmask_t mask;
+	odp_schedule_group_t group;
+
+	odp_thrmask_zero(&mask);
+	odp_thrmask_set(&mask, odp_thread_id());
+	group = odp_schedule_group_create("create_group", &mask);
+
+	CU_ASSERT_FATAL(group != ODP_SCHED_GROUP_INVALID);
+
+	test_group_basic(group);
+}
+
+static void scheduler_test_group_param_init(void)
+{
+	odp_schedule_group_param_t param;
+
+	memset(&param, 0xff, sizeof(param));
+	odp_schedule_group_param_init(&param);
+	CU_ASSERT(param.cache_stash_hints.common.regions.all == 0);
+	CU_ASSERT(param.cache_stash_hints.num == 0);
+}
+
+static void scheduler_test_create_group_2(void)
+{
+	odp_schedule_group_param_t param;
+	odp_thrmask_t mask;
+	odp_schedule_group_t group;
+
+	odp_thrmask_zero(&mask);
+	odp_thrmask_set(&mask, odp_thread_id());
+	odp_schedule_group_param_init(&param);
+	param.cache_stash_hints.common.regions.event_user_area_l2 = 1;
+	param.cache_stash_hints.common.regions.event_data_l2 = 1;
+	param.cache_stash_hints.common.event_user_area.l2.offset = 0;
+	param.cache_stash_hints.common.event_user_area.l2.len = ODP_CACHE_LINE_SIZE;
+	param.cache_stash_hints.common.event_data.l2.offset = 0;
+	param.cache_stash_hints.common.event_data.l2.len = ODP_CACHE_LINE_SIZE;
+	group = odp_schedule_group_create_2("create_group", &mask, &param);
```

Comment:
It would be good to fill in something to all 8 stash targets. So that in minimum we check that those bits/fields are there in the struct and create_2 does not return when filling those.